changes/6920.fixed

```diff
@@ -0,0 +1,3 @@
+Added log messages to help troubleshoot failures when rendering UI Component Framework `extra_buttons`.
```

Comment:
Should this be a separate changelog fragment?

Reply:
Were you thinking `.added` for this? I consider it a "fix" rather than a "feature", but I'm persuadable.

Reply:
Yeah, I was thinking `.added` for this. It is not technically something that was `missing` but an enhancement to the previous iteration. Definitely not blocking.